com.microsoft.mrtk.extendedassets/Models/Model_PushButton.fbx.meta

```diff
@@ -1,26 +1,14 @@
 fileFormatVersion: 2
-guid: 2e4e683eccb3efe49ba775ada36cbc8f
```

Comment:
Oh, what's the change here? Did we have Model_PushButton.fbx in both Standard and Extended? Hopefully they were the same 😬

Reply:
Also, I _could_ see a button type being considered "standard" imo

Reply:
we did have two copies. the models were identical and the guids different.

the interesting thing is that the round button was in both places and the square in extended assets

Reply:
Oooh the square already being in extended assets is good enough signal for me 😜